build/infrastructure/eo/chart/values.yaml

```diff
@@ -4,7 +4,7 @@ eo-base-helm-chart:
       replicaCount: 2
       image:
         repository: ghcr.io/energinet-datahub/eo-frontend-app
-        tag: 0.2.25
+        tag: 0.2.26
```

Comment:
Please revert this, updating EO should only be necessary when changing in shared functionality across datahub and energioprindelse like the design system (watt)